src/azure-openapi-validator/index.ts

```diff
@@ -1,9 +1,14 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
 import { nodes } from "jsonpath";
 import { Message } from "../jsonrpc/types";
 import { rules } from "./rule";
 
 // register rules
 require("./rules/SecurityDefinitionsStructureValidation");
+require("./rules/DescriptionMustNotBeNodeName");

```

Comment:
being new to this... O see that the rules get executed via adding a plugin to AutoRestPluginHost (https://github.com/olydis/azure-openapi-validator/blob/master/src/index.ts#L10). If we publish this as an npm package, should autorest a dependency? I'm probably missing something and may need to better understand how the architecture of the plugins is.